// 武将碎片
message HeroChipPB{
	required int32 hero_chip_no = 1;  //编号
	required int32 hero_chip_num = 2;  //数量
}